src/components/Header/GroupHead/GroupHead.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { CATE_MENU } from './CATE_MENU.js';
+import Common from '../../../components/Common/Common';
+import styled from 'styled-components';
```

Comment:
```suggestion
import React from 'react';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import Common from '../../../components/Common/Common';
import { CATE_MENU } from './CATE_MENU.js';
```

import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)

Reply:
수정했습니다~! 확인 부탁드립니다 :)

다만 한가지 안되고 있는게 있어서 문의 드립니다'
콘솔에 하단과 같은 경고 문구가 뜨는데 어떻게 수정해야할까요?

<img width="1341" alt="스크린샷 2023-03-28 오후 1 43 28" src="https://user-images.githubusercontent.com/89458455/228130427-6adfd718-12a2-41c8-a622-1023114bf338.png">